services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersValidateSimpleFlowSpec.groovy

```diff
@@ -0,0 +1,517 @@
+        def createdMetersIdSrcSwitch = getCreatedMetersId(srcSwitch.dpId, defaultMetersSrcSwitch)
+        def createdMetersIdDstSwitch = getCreatedMetersId(dstSwitch.dpId, defaultMetersDstSwitch)
+        def burstSizeSrcSwitch = switchHelper.getExpectedBurst(srcSwitch.dpId, flow.maximumBandwidth)
+        def burstSizeDstSwitch = switchHelper.getExpectedBurst(dstSwitch.dpId, flow.maximumBandwidth)
+
+        createdMetersIdSrcSwitch.size() == 1
+        createdMetersIdDstSwitch.size() == 1
+
+        and: "The correct info is stored in the validate meter response"
+        def meterValidateInfoSrcSwitch = northbound.validateMeters(srcSwitch.dpId)
+        def meterValidateInfoDstSwitch = northbound.validateMeters(dstSwitch.dpId)
+        def createdCookieSrcSwitch = getCreatedCookies(srcSwitch.dpId, createdMetersIdSrcSwitch)
+        def createdCookieDstSwitch = getCreatedCookies(dstSwitch.dpId, createdMetersIdDstSwitch)
+
+        def createdCookies = createdCookieSrcSwitch + createdCookieDstSwitch
+
+        meterValidateInfoSrcSwitch.meters.proper.collect { it.meterId }.containsAll(createdMetersIdSrcSwitch)
+        meterValidateInfoDstSwitch.meters.proper.collect { it.meterId }.containsAll(createdMetersIdDstSwitch)
+
+        meterValidateInfoSrcSwitch.meters.proper.collect { it.cookie }.containsAll(createdCookieSrcSwitch)
+        meterValidateInfoDstSwitch.meters.proper.collect { it.cookie }.containsAll(createdCookieDstSwitch)
+
+        [meterValidateInfoSrcSwitch, meterValidateInfoDstSwitch].each {
+            it.meters.proper.each {
+                assert it.rate == flow.maximumBandwidth
+                assert it.flowId == flow.id
+            }
+        }
+
+        meterValidateInfoSrcSwitch.meters.proper.burstSize[0] == burstSizeSrcSwitch
+        meterValidateInfoDstSwitch.meters.proper.burstSize[0] == burstSizeDstSwitch
+
+        [meterValidateInfoSrcSwitch, meterValidateInfoDstSwitch].each {
+            it.meters.proper.every {
+                ["KBPS", "BURST", "STATS"].containsAll(it.flags)
+            }
+        }
+
+        [srcSwitch, dstSwitch].each {
+            checkNoMeterValidation(it.dpId, ["proper"])
+        }
+
+        and: "Created rules are stored in the proper section"
+        [meterValidateInfoSrcSwitch, meterValidateInfoDstSwitch].each {
+            it.rules.properRules.containsAll(createdCookies)
+        }
+
+        and: "The rest fields in the rule section are empty"
+        [srcSwitch, dstSwitch].each {
+            checkNoRuleMeterValidation(it.dpId, ["properRules"])
+        }
+
+        then: "Delete the flow"
+        flowHelper.deleteFlow(flow.id)
+
+        and: "Check that meter validate info is also deleted"
+        [srcSwitch, dstSwitch].each {
+            checkNoRuleMeterValidation(it.dpId)
+            checkNoMeterValidation(it.dpId)
+        }
+    }
+
+    def "Able to move meter info into the misconfigured section"() {
```

Comment:
Not the purpose of the test.
`Switch validation is able to detect misconfigured meters` sounds closer to what we actually test

Reply:
Fixed